burst/deploy/burst-dev.yaml

```diff
@@ -39,6 +39,8 @@ metadata:
 rules:
 - apiGroups:
   - autoscaling
+  - apps
+  - ""
```

Comment:
empty string?

Reply:
Apparently it corresponds to the core API group `v1/*`. Seems weird but got it from the documentation at https://kubernetes.io/docs/reference/access-authn-authz/rbac/#role-example